### PR TITLE
support for terraform 0.13

### PR DESCRIPTION
**Pull request by Maciej-Gierwatowski**

### CODE REVIEW

**Comment by bmwosz** [comment]
AFAIK we wanted to close this PR without merging to not work with already depricated stuff

File: main.tf [hunk 1]
```diff
@@ -10,8 +10,7 @@ module "label" {
 
 # Main cluster's Security Groups
 module "security" {
-  source = "git@github.com:BerlingskeMedia/bm.terraform-module.security?ref=production"
-  //source      = "../bm.terraform-module.security"
+  source      = "git@github.com:BerlingskeMedia/bm.terraform-module.security?ref=production"
```
Comment (bmtoml) [inline main.tf:13]: " we are not, In 12 PR there is no production tag in main.tf file"

ghosts then?

**Review by bmtoml — CHANGES_REQUESTED** [review]

**Review by bmtoml — CHANGES_REQUESTED** [review]
maciej, close this?

**Comment by wszychta** [comment]
This PR is useless for us now.

**Pull request closed without merging**